mep/people/views.py

```diff
@@ -244,6 +249,29 @@ def get_context_data(self, **kwargs):
                 'endDate': end.isoformat()
             } for start, end in account.event_date_ranges]
         }
+
+        # plottable locations for member address map visualization
+        # addresses can be stored on either Person or Account
+        addresses = Address.objects.filter(Q(account__pk=account.pk) |
+            Q(person__pk=self.object.pk)) \
+            .filter(location__latitude__isnull=False) \
+            .filter(location__longitude__isnull=False)
+        context['addresses'] = [
```

Comment:
I assume you're converting from queryset to list of dict so that you can pass it to leaflet - probably better to comment at the beginning of the block to make that explicit (your note about the lat/long makes it implicit).

Why aren't lat/long serializable?

Might be preferable to use format on the lat/long so we can set desired precision.

Reply:
Actually, it might also make sense to write a queryset method that returns the data in the format we need it for the map (although possibly hold off on that until we refactor for the bigger map).  I think we can probably leverage queryset annotation and values to get django to generate it in this format for us.

Reply:
they aren't serializable because they are `Decimal`s (or at least that's the error I got when using `json.dumps`.). i think it's fine to just use as much precision as we have; can't imagine a case where more is bad?

i think it's OK to hold off until we refactor for the bigger map. i went this route of doing everything in the view because we're taking the same approach with aggregating/formatting the data for the timeline.